source/reactor/inc/reactor/EventHandlers.h

```diff
@@ -0,0 +1,68 @@
+#ifndef Reactor_EventHandlers_H
+#define Reactor_EventHandlers_H
+
+#include <functional>
+
+namespace u {
+namespace reactor {
+class EventLoop;
+
+/**
+ * @brief Event handler function definition.
+ *
+ * Parameters to the event handler function are:
+ * - The file descriptor of the OS resource where the event was originated at. Event handler can
```

Comment:
Since this is an actual function, you can document it with
```suggestion
 * @param[in] fd The file descriptor of the OS resource where the event was originated at.
```
(of course, it requires naming the parameters in the function signature).